src/test/java/com/microsoft/azure/kusto/kafka/connect/sink/FileWriterTest.java

```diff
@@ -80,7 +80,7 @@ public void testGzipFileWriter() throws IOException {
 
         HashMap<String, Long> files = new HashMap<>();
 
-        final int MAX_FILE_SIZE = 128;
+        final int MAX_FILE_SIZE = 100;
```

Comment:
not that it matters much, but why not a power of 2?

Reply:
just chose a number that is less then 2 messages - don't see why it matters here that it will be  a power of 2